java/maven.refactoring/src/org/netbeans/modules/maven/refactoring/MavenRefactoringPlugin.java

```diff
@@ -73,6 +93,46 @@ class MavenRefactoringPlugin implements RefactoringPlugin {
             }
             @Override public void cancel() {}
         };
+        
+        if (refactoring != null) {
+            ModelOperation<POMModel> operation = (final POMModel model) -> {
```

Comment:
just a thought: if `operation` is renamed to `renameMainClassProp`, it would implicitly document what it does.

Reply:
Sure done